src/Interpreters/SystemLog.h

```diff
@@ -79,6 +82,8 @@ struct SystemLogs
     std::shared_ptr<ProcessorsProfileLog> processors_profile_log;
 
     std::vector<ISystemLog *> logs;
+
+    Poco::Logger * sys_log = &Poco::Logger::get("SystemLogs");
```

Comment:
As logging removed now, need to remove this line and the include with logger_useful.h